### PR TITLE
1.x dev

### PR DESCRIPTION
**Pull request by lordrhodos**
This PR is the next one in an attemp to move the `starlit/*` packages to a `1.0.0` release

### CODE REVIEW

File: tests/TestDbEntity.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Starlit\Db;
```
Comment (felixsand) [inline tests/TestDbEntity.php]: Not sure about this namespace here?
Maybe it should also be moved to say `tests/resources/TestDbEntity.php` or something like that instead?

**Review by felixsand — CHANGES_REQUESTED** [review]
Looks good! 👍 
Just one question regarding the TestDbEntity class

File: tests/TestDbEntity.php (hunk 1 above)
Comment (lordrhodos) [reply tests/TestDbEntity.php]: I agree. Maybe we should think about using dedicated namespaces for the tests in general, e.g. `Starlit\Db\Tests` ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## [Unreleased]
+### Added
+- add changelog
+
+### Changed
+- add strict_types and return type declarations
+- update PHP 7.3 to Travis CI configuration
+- update Travis CI to run tests on PHP 7.3
+- update phpunit/phpunit package to
```
Comment (felixsand) [inline CHANGELOG.md]:
```suggestion
- update phpunit/phpunit package to ^7.5
```

**Review by felixsand — CHANGES_REQUESTED** [review]
Just noticed a typo. Otherwise this is ready I think :)

**Review by felixsand — APPROVED** [review]

**Pull request merged**